sys/net/application_layer/uhcp/uhcpc.c

```diff
@@ -38,6 +38,10 @@ void uhcp_client(uhcp_iface_t iface)
 
     /* create listening socket */
     int res = sock_udp_create(&sock, &local, NULL, 0);
+    if (res < 0) {
+        puts("uhcp_client(): cannot create listening socket");
```

Comment:
puts, not LOG_ERROR or something?

Reply:
I wanted to use a LOG_ function but puts is already used below, so went for consistency with other message displays.

Reply:
ok

Reply:
Thanks!